docs/specs/clients/notify/client-sdk-api.md

```diff
@@ -56,17 +58,32 @@ abstract class Client {
     encryptedMessage: string,
   }): Promise<NotifyMessage>;
   
+  // Generates and stores an identity key, indexed via the following key
```

Comment:
I'm not sure that we're need to store identity key before it was registered

Reply:
We will have to add a flag into local storage that identity is not register on server or something. But we need to generate the key before we generate the message. And we need the message to register the identity key in keys server

Reply:
Agree that we should generate it here, but should we store it ..

Reply:
If we consider returning the identity key instead of storing it, could we store it in-memory rather than localstorage instead?

Reply:
@llbartekll we decided that the `prepareRegistration` returns `cacao.payload`, `identityPrivateKey` and `siwe message` to sign. Later on `register` we need that `cacao.payload` to verify the signature and `identityPrivateKey` to recreate `identityPublicKey` and register it in Keys Server